tests/simple/var.uplc

```diff
@@ -1 +1 @@
-(program 1.0.0 x)
```

Comment:
I think we should keep this test `(program 1.0.0 x)`

Reply:
Same as force and delay.